src/TConsole.cpp

```diff
@@ -453,13 +437,13 @@ TConsole::TConsole(Host* pH, ConsoleType type, QWidget* parent)
     mpBufferSearchBox->setFont(mpHost->mCommandLineFont);
     mpBufferSearchBox->setFocusPolicy(Qt::ClickFocus);
     mpBufferSearchBox->setPlaceholderText("Search ...");
-    QPalette __pal;
```

Comment:
:boom: An identifier ***containing*** a double underscore is, apparently, **forbidden** by C++ identifier rules. https://stackoverflow.com/a/228797/4805858 ...